### PR TITLE
DP-193 - Identifier Uri mapping added

### PR DESCRIPTION
**Pull request by maciej-goaco**

### CODE REVIEW

File: Services/CO.CDP.OrganisationInformation/CO.CDP.OrganisationInformation.csproj [hunk 1]
```diff
@@ -8,4 +8,8 @@
         <NoWarn>$(NoWarn);1591</NoWarn>
     </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="8.0.0" />
+    </ItemGroup>
```
Comment (jakzal) [inline Services/CO.CDP.OrganisationInformation/CO.CDP.OrganisationInformation.csproj:13]: Why is it needed here?

Comment (maciej-goaco) [reply Services/CO.CDP.OrganisationInformation/CO.CDP.OrganisationInformation.csproj]: it's not, removed.

File: Services/CO.CDP.Organisation.WebApi/IIdentifierService.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+namespace CO.CDP.Organisation.WebApi;
+
+public interface IIdentifierService
+{
+    string? GetRegistryUri(string? scheme, string? identifierId);
```
Comment (jakzal) [inline Services/CO.CDP.Organisation.WebApi/IIdentifierService.cs:5]: Why are `scheme` and `identifierId` nullable?

File: Services/CO.CDP.Organisation.WebApi/appsettings.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
       }
     }
   },
+  "OrganisationApiHostUrl": "http://localhost:8082",
```
Comment (jakzal) [inline Services/CO.CDP.Organisation.WebApi/appsettings.json:11]: "HostUrl" doesn't make much sense - it's either one or the other. Can we name the option `OrganisationApiUrl` or `OrganisationWebApiUrl` to be consistent with `OrganisationAppUrl` we already have?

File: Services/CO.CDP.Organisation.WebApi/IIdentifierService.cs (hunk 2 above)
Comment (maciej-goaco) [reply Services/CO.CDP.Organisation.WebApi/IIdentifierService.cs]: Scheme previously was and not anymore with current usage, so changed that.
IdentifierId seems to be made purposely nullable, even though not sure what value such entry would bring.

File: Services/CO.CDP.Organisation.WebApi/appsettings.json (hunk 3 above)
Comment (maciej-goaco) [reply Services/CO.CDP.Organisation.WebApi/appsettings.json]: done

**Review by rmohammed-goaco — APPROVED** [review]

**Pull request merged**